### PR TITLE
[graphql/rpc] tx sigs

### PR DESCRIPTION
**Pull request by stefan-mysten**
## Description 

Add transaction signatures as a base64 value. 

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: crates/sui-graphql-rpc/src/context_data/db_data_provider.rs [hunk 1]
```diff
@@ -1256,6 +1256,16 @@ impl TryFrom<StoredTransaction> for TransactionBlock {
         };
 
         let kind = TransactionBlockKind::from(sender_signed_data.transaction_data().kind());
+        let signatures = sender_signed_data
+            .tx_signatures()
+            .iter()
+            .map(|s| {
+                Some(crate::types::transaction_signature::TransactionSignature {
```
Comment (wlmyng) [inline crates/sui-graphql-rpc/src/context_data/db_data_provider.rs:1263]: Can we import the type

File: crates/sui-graphql-rpc/src/context_data/db_data_provider.rs [hunk 2]
```diff
@@ -1256,6 +1256,16 @@ impl TryFrom<StoredTransaction> for TransactionBlock {
...
+                    base64_sig: fastcrypto::encoding::Base64::from_bytes(s.as_ref()).encoded(),
```
Comment (wlmyng) [inline crates/sui-graphql-rpc/src/context_data/db_data_provider.rs:1264]: Let's use crate Base64

File: crates/sui-graphql-rpc/src/types/transaction_signature.rs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use async_graphql::SimpleObject;
+
+// TODO: flesh out the scalar transaction signature
+#[derive(SimpleObject, Clone, Eq, PartialEq)]
+pub(crate) struct TransactionSignature {
+    pub base64_sig: String,
```
Comment (wlmyng) [inline crates/sui-graphql-rpc/src/types/transaction_signature.rs:9]: Proposal to return these as base64 type

File: crates/sui-graphql-rpc/src/context_data/db_data_provider.rs (hunk 1 above)
Comment (stefan-mysten) [reply crates/sui-graphql-rpc/src/context_data/db_data_provider.rs:1263]: Oops, sorry about that.

**Review by wlmyng — APPROVED** [review]

**Pull request merged**